docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -756,7 +765,7 @@ Step 5: System Configuration
 
      passwd
 
-#. Enable importing bpool
+#. Enable importing bpool (Of cource, if you created them)
```

Comment:
Let's drop it, if user did something not by default - user should understand consequences.

Reply:
My comment is still actual